list.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'enumerable'
+
+class MyList
+  include MyEnumerable
+  attr_reader :list
+
+  def initialize(list)
+    @list = list
+  end
```

Comment:
- [ ] Kindly be noticed that the `#initialize` method accepts multiple arguments, thanks to that, calling the constructor with a list of numbers as in the test code is possible `MyList.new(1, 2, 3, 4)`.

```ruby
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
end
```

Reply:
Done. Thank you for your help.